.github/workflows/hygiene-check.yml

```
@@ -0,0 +1,173 @@
+          fetch-depth: 0
+          
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+          
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install PyGithub openai anthropic pyyaml requests
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)